app/src/main/kotlin/com/saltedge/authenticator/features/settings/list/SettingsListViewModel.kt

```diff
@@ -84,6 +89,17 @@ class SettingsListViewModel(
                 itemIsClickable = true
             )
         )
+        val darkModeItem = SettingsItemViewModel(
```

Comment:
System dark mode selector should switch to disabled state if was clicked custom dark mode switch in MainActivity